react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -426,6 +467,38 @@ class Thumbnail extends Component<Props, State> {
         if (prevState.displayMode !== this.state.displayMode) {
             this._onDisplayModeChanged();
         }
+
+        // TODO: after converting this component to a react function component,
+        // use a custom hook to update local track streaming status.
+        const { _videoTrack, dispatch, _sourceNameSignalingEnabled } = this.props;
+
+        if (_sourceNameSignalingEnabled &&
+            prevProps._videoTrack?.jitsiTrack?.getSourceName?.() !== _videoTrack?.jitsiTrack?.getSourceName?.()) {
```

Comment:
same as above no need for `?` in `.getSourceName?.()`

Reply:
changed